.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,58 @@
+name: Smoke Test
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  detect-changes:
+    uses: ./.github/workflows/detect-changes-matrix.yml
```

Comment:
shouldn't this be the opposite? as the smoke tests is for changes in core rather than in integrations

Reply:
This is me making a small AC preparation crime.
Wanted to create a convention where you'd add something according to convention to run the same smoke test for an integration later.

The detect changes also detects changes in core